src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadTaxonomiesData.php

```diff
@@ -99,9 +99,10 @@ protected function createTaxonomy($code, array $taxonomyName, array $taxonsArray
                     $this->setReference('Sylius.Taxon.'.$taxonName, $taxon);
                 }
             }
-            $taxonomy->addTaxon($taxon);
+
+            $rootTaxon->addChild($taxon);
```

Comment:
does this work? It looks like it's adding child to $rootTaxon which was set in a previous loop... Wouldn't this add ALL child taxons to whichever root was set last in the previous loop?

Maybe I'm reading it wrong, but if it is broken, then it was from before these changes.